example_profile.mobileconfig

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PayloadContent</key>
+	<array>
+		<dict>
+			<key>PayloadDisplayName</key>
+			<string>Nudge Configuration</string>
+			<key>PayloadIdentifier</key>
+			<string>com.erikng.nudge</string>
+			<key>PayloadType</key>
+			<string>com.erikng.nudge</string>
+			<key>PayloadUUID</key>
+			<string>41AF5F92-8C0F-40B2-B6A6-09899F3D23A2</string>
+			<key>PayloadVersion</key>
+			<integer>1</integer>
+			<key>button_title_text</key>
+			<string>Ready to start the update?</string>
+			<key>button_sub_titletext</key>
+			<string>Click on the button below.</string>
+			<key>cut_off_date</key>
+			<string>2019-12-31-00:00</string>
+			<key>cut_off_date_warning</key>
+			<integer>14</integer>
+			<key>time_between_notifications</key>
+			<integer>0</integer>
+			<key>dismissal_count_threshold</key>
+			<integer>10</integer>
+			<key>logo_path</key>
+			<string>/path/to/company_logo.png</string>
+			<key>main_subtitle_text</key>
+			<string>A friendly reminder from your local IT team</string>
+			<key>main_title_text</key>
+			<string>macOS Update</string>
+			<key>minimum_os_version</key>
+			<string>10.14.6</string>
```

Comment:
Similar to L53 below, maybe we change this to `10.15.6` and L39 below to `19G2021`?

Reply:
fixed in https://github.com/macadmins/nudge/pull/49/commits/ab7b5a8062b358c4bf8fa3d5d1d80d360e25e265